packages/flutter_tools/lib/src/commands/channel.dart

```diff
@@ -17,6 +19,12 @@ class ChannelCommand extends FlutterCommand {
       help: 'Include all the available branches (including local branches) when listing channels.',
       hide: !verboseHelp,
     );
+    argParser.addFlag(
+      'cache-artifacts',
+      help: 'After switching channels, download all required binary artifacts. '
+            'This is the equivalent of running "flutter precache" with the "--all" flag.',
```

Comment:
Possible typo; I don't think `--all` is a real option.

```suggestion
            'This is the equivalent of running "flutter precache" with the "--all-platforms" flag.',
```

Reply:
Ah never mind I see you added an alias

Reply:
It's now an alias: https://github.com/flutter/flutter/pull/118129/files#diff-e95a30630956e29a2d852d06aeccedf10ab830951e95bd6f42c9624dd7baf5a2R29

Reply:
I committed your change however